website/docs/_Mining/guide.mdx

```diff
@@ -0,0 +1,97 @@
+
+## Step 4 Select a Mining Pool[](https://nervos-ckb-docs-git-v1-cryptape.vercel.app/docs/basics/guides/mining-ckb#step-4---select-a-mining-pool)
+
+Selecting a mining pool is one of the most important decisions you will make when you start mining. This is not only important for your profitability as a miner, but it is also important for the long-term health of the network itself.
+
+> Decentralized blockchains can become more vulnerable to a 51% attack when too much mining power is concentrated in single mining pools. When choosing a pool, consider selecting one of the smaller pools to help keep the hash power decentralized.
+> 
+
+Larger pools tend to find blocks more quickly, and thus have faster payouts for miners. However, the payouts are smaller since you represent a smaller slice of the total pool. Smaller pools will have slower payouts but each payout will be larger. In theory, total earnings between a large and small pool should be roughly the same in the long term.
+
+A few other things to consider when selecting a pool:
+
+- **Mining Pool Fees**: Mining pool fees normally hover between 1% and 3%. In addition to the percentage, there are also different [payment systems](https://medium.com/luxor/mining-pool-payment-methods-pps-vs-pplns-ac699f44149f) to consider.
+- **Transparency**: Most pools have metrics and charts available that give users the ability to see how the pool is performing. A dishonest pool could claim to have a low fee and then take an unfair share of the mining rewards. The more transparent the pool is, the less likely this is to occur.
+- **Security**: A well-run pool should have some history and a track record of successful payouts. Remember, you are trusting the pool with your earnings. If they have poor security, they may lose some of your funds.
+- **Proximity**: Pools that are closer to your location are usually better. This is because PoW is inherently a race to find answers. The closer you are to a pool, the faster you can communicate, and this can result in slightly higher payouts.
+
+Part of a miner's duty is to protect the integrity of the network. A successful [51% attack](https://99bitcoins.com/51-percent-attack/) on the network would likely result in lower profitability for all miners. This is why we stress so thoroughly that you perform your due diligence when selecting a pool and do your part to promote decentralization by supporting smaller pools.
+
+The steps to create a pool account and the settings needed for your miner can differ from pool to pool. This information is specific to each pool and provided by the pools themselves. Search for specific mining guides for your chosen mining pool:
+
+- F2Pool: https://f2pool.zendesk.com/hc/en-us/articles/22893426126489-How-to-mine-Nervos
+- Antpool: https://antpoolsupport-hc.zendesk.com/hc/en-us/articles/5985104597913-CKB-Mining
+- Binance Pool: https://antpoolsupport-hc.zendesk.com/hc/en-us/articles/5985104597913-CKB-Mining
```

Comment:
This link should be: https://www.binance.com/en/support/faq/how-to-mine-nervos-ckb-on-binance-pool-6967ad3306b84ec49dcbb5ed8ac30a9b

Reply:
thanks! updated